src/backend/index.js

```diff
@@ -60,7 +61,22 @@ async function invalidateFeed(feedData) {
  * Process all of these Feed objects into Redis and the feed queue.
  * @param {Array<Feed>} feeds - the parsed feed Objects to be processed.
  */
-function processFeeds(feeds) {
+async function processFeeds(feeds) {
+  const redisFeedIds = await getFeeds();
```

Comment:
I'm skeptical that this algorithm will work.  At the very least, this needs some comments to help the reader understand your intent.  Walk me through what you expect this to do in comments.  Then let's see if you're right.

Reply:
I made some updates to the logic, as well as added additional comments in https://github.com/Seneca-CDOT/telescope/pull/727/commits/9060e955f1acb68c81e23bc0caeef71627db583f. I'll also lay out my understanding of the logic here:

1. Here, I gather the two sets of feed `id`s (from Redis and the Wiki feed list, respectively) in preparation for determining the difference between the two sets:
   https://github.com/Seneca-CDOT/telescope/blob/9060e955f1acb68c81e23bc0caeef71627db583f/src/backend/index.js#L68-L74
1. Here, all [`deletedFeeds`](https://github.com/Seneca-CDOT/telescope/issues/294#issuecomment-580372724) are removed from Redis:
   https://github.com/Seneca-CDOT/telescope/blob/9060e955f1acb68c81e23bc0caeef71627db583f/src/backend/index.js#L75-L84
   - (`removeFeedById()` was updated in response to https://github.com/Seneca-CDOT/telescope/pull/727#discussion_r379869371) 
1. Finally, the entirity of the Wiki feed list (including all [`newFeeds`](https://github.com/Seneca-CDOT/telescope/issues/294#issuecomment-580372724)) proceeds to be processed:
   https://github.com/Seneca-CDOT/telescope/blob/9060e955f1acb68c81e23bc0caeef71627db583f/src/backend/index.js#L89-L97

This represents the synchronization process, as I understand it. As mentioned in my original post, however, I do not yet know whether `processFeeds()` is the most best place to contain this process.